ts/Series/PolarComposition.ts

```diff
@@ -1422,17 +1422,24 @@ function wrapPointPos(
     chartCoordinates?: boolean,
     plotY: number|undefined = this.plotY
 ): [number, number]|undefined {
-    const { plotX, series } = this,
-        { chart } = series;
-
-    if (chart.polar && !this.destroyed && isNumber(plotX) && isNumber(plotY)) {
-        return [
-            plotX + (chartCoordinates ? chart.plotLeft : 0),
-            plotY + (chartCoordinates ? chart.plotTop : 0)
-        ];
-    }
+    if (!this.destroyed) {
+        const { plotX, series } = this,
+            { chart } = series;
 
-    return proceed.call(this, chartCoordinates, plotY);
+        if (
+            chart.polar &&
+            !this.destroyed &&
```

Comment:
This condition is now redundant, isn't it? Since we're already checking `this.destroyed` on line 1425.